Neos.Flow/Classes/I18n/Translator.php

```diff
@@ -126,19 +126,29 @@ public function translateByOriginalLabel($originalLabel, array $arguments = [],
         if ($locale === null) {
             $locale = $this->localizationService->getConfiguration()->getCurrentLocale();
         }
-        $pluralForm = $this->getPluralForm($quantity, $locale);
 
-        $translatedMessage = $this->translationProvider->getTranslationByOriginalLabel($originalLabel, $locale, $pluralForm, $sourceName, $packageKey);
-
-        if ($translatedMessage === false) {
-            $translatedMessage = $originalLabel;
-        }
-
-        if (!empty($arguments)) {
-            $translatedMessage = $this->formatResolver->resolvePlaceholders($translatedMessage, $arguments, $locale);
+        foreach ($this->localizationService->getLocaleChain($locale) as $localeInChain) {
+            $translatedMessage = $this->translationProvider->getTranslationByOriginalLabel(
+                $originalLabel,
+                $localeInChain,
+                $this->getPluralForm($quantity, $localeInChain),
+                $sourceName,
+                $packageKey
+            );
+
+            if ($translatedMessage !== false) {
+                return $arguments === []
+                    ? $translatedMessage
+                    : $this->formatResolver->resolvePlaceholders(
+                        $translatedMessage,
+                        $arguments,
+                        $localeInChain
+                    )
+                ;
+            }
         }
 
-        return $translatedMessage;
+        return $originalLabel;
```

Comment:
I am currently not sure if possible placeholders in the `$originalLabel` should be resolved as well. Before they were, so maybe now at least resolve them using the given (or current) locale?

This line could then be changed to
```php
    return $arguments === []
        ? $originalLabel
        : $this->formatResolver->resolvePlaceholders($originalLabel, $arguments, $locale)
    ;
```